source/change-streams/tests/unified/change-streams-disambiguatedPaths.yml

```diff
@@ -0,0 +1,102 @@
+description: "disambiguatedPaths"
+schemaVersion: "1.9"
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+
+runOnRequirements:
+  - minServerVersion: "6.1.0"
+    topologies: [ replicaset, sharded-replicaset, load-balanced ]
```

Comment:
Do these new tests pass on sharded clusters + serverless? I would hope so, but we've had issues in the past with change streams tests on those topologies.

Reply:
They pass on sharded clusters - https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_next_ubuntu_18.04_erbium_test_latest_sharded_cluster_patch_f28ba0de3bc925267a185965eb971ecf91bc2680_62fd236be3c3313be42d354a_22_08_17_17_20_44/0?type=T#L835

Reply:
> They don't run on serverless

The spec says:

> Unified spec tests from all specifications MUST be run against Atlas Serverless.

Thanks for adding `sharded`; it's sort of a strange topology, but might as well add.